HW-2020-04-30/src/Lists/List2/Names.java

```diff
@@ -0,0 +1,18 @@
+package Lists.List2;
+
+import java.util.List;
+
+/*2.Есть список с именами: ​Ivan, Maria, Stephan, John, Amalia​.
+        Написать функцию,которая вернет список, в котором не содержатся имена исходного списка,длиной 4*/
+public class Names {
+
+    public static List<String> length4 (List<String> list) {
+        for(int i = 0; i < list.size(); i++) {
+            if(list.get(i).length()==4) {
+                list.remove(i);
+                i--; /*Need to do this because after remove we skip one number in counter. Also did this with Iterator.*/
```

Comment:
I remember I was already commenting on this code. Why it is in this PR again?

Reply:
my bad, uploaded with old commits. Will check how to upload only single commit from new branch.